src/connection.rs

```diff
@@ -225,9 +216,12 @@ impl Connection {
         self.write_continuation = None;
         self.write_queue.clear();
         self.is_connected = false;
-        let timeout = ServerTimeout::Reconnect(self.token);
+        let timeout = ServerTimeout::Reconnect(token);
         let handle = event_loop.timeout_ms(timeout, duration).unwrap();
-        Ok((duration, timeout, handle))
+
+        info!("{:?}: {:?} reset, will attempt to reconnect in {}ms", self,
```

Comment:
One of the things I tried to do is limit the amount of logging coming out of connection.rs, in lew of doing it in server.rs.  Reason being that it's useful to include the server ID in the log messages in the case where a single processes is running multiple Raft servers.